src/pages/HomePage.jsx

```diff
@@ -13,19 +13,27 @@ import { globalStore } from "../stores";
 export const HomePage = () => {
   const { posts } = globalStore.getState();
 
+  const user = globalStore.getState().currentUser;
```

Comment:
```suggestion
const { posts, currentUser: user } = globalStore.getState();
```

요런 스타일로 축약해서 적어도 괜찮을 것 같아요 ㅎㅎ